QUICK-START.md

```diff
@@ -78,11 +80,20 @@ release package.
 Populate the `Host`, `BootstrapUsername`, `BootstrapPassword`, and `BootstrapTenant` fields with the
 corresponding values for your Cumulocity tenant. The `Host` field should be populated with the
 MQTT broker's host for your Cumulocity instance, for example, `mqtt.eu-latest.cumulocity.com` or
-`mqtt.us-latest.cumulocity.com`. Do not include the scheme or port as they are automatically 
-selected to be https:// and 8883.
+`mqtt.us-latest.cumulocity.com`. Do not include the scheme or port as they are automatically selected to be https:// and 8883.
 
 ![Connector Configuration File Bootstrap Edit](images/ConnectorConfigBootstrap.png)
 
+Typically, the following example values are the standard values for all public Cumulocity instances maintained by SoftwareAG on its **European** server,
+
+The `BootstrapTenant` is the bootstrap tenantID which is `management`;
+
+The `BootstrapUsername` is the bootstrap username, which is `devicebootstrap`;
+
+The `Host` is `mqtt.eu-latest.cumulocity.com`;
+
+And the `BootstrapPassword` is `Fhdt1bb1f`.
+
```

Comment:
Hi Hepu,

Are we able to add a section for the US American mqtt.us-latest.cumulocity.com server as well?

The credentials should be the same, just a different host: https://tech.forums.softwareag.com/t/mqtt-static-credentials/203829/3